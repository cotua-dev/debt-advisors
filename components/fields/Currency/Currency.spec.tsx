import { render, RenderResult } from '@testing-library/react';
import { Currency } from './index';

describe('Currency Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Currency/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
