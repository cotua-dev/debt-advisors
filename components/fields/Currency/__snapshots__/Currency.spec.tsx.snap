// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Currency Component renders 1`] = `
<div>
  <strong>
    Currency
  </strong>
</div>
`;
